cmdline/bin/dcat.dart

```diff
@@ -36,34 +36,35 @@ void main(List<String> arguments) {
   dcat(paths, argResults[LINE_NUMBER]);
 }
 
-Future dcat(List<String> paths, bool showLineNumbers) {
+Future dcat(List<String> paths, bool showLineNumbers) async {
```

Comment:
Our tools can see `async` and infer that the function will return a Future. I think our style guide can ask our developers not to include Future when they already included async in the function/method signature.